x-pack/test/security_solution_api_integration/package.json

```diff
@@ -5,181 +5,199 @@
+    
+    "prebuilt_rules_large_prebuilt_rules_package:server:serverless": "npm run initialize-server:rm prebuilt_rules/large_prebuilt_rules_package serverless",
+    "prebuilt_rules_large_prebuilt_rules_package:runner:serverless": "npm run run-tests:rm prebuilt_rules/large_prebuilt_rules_package serverless serverlessEnv",
+    "prebuilt_rules_large_prebuilt_rules_package:qa:serverless": "npm run run-tests:rm prebuilt_rules/large_prebuilt_rules_package serverless qaEnv",
+    "prebuilt_rules_large_prebuilt_rules_package:server:ess": "npm run initialize-server:rm prebuilt_rules/large_prebuilt_rules_package ess",
+    "prebuilt_rules_large_prebuilt_rules_package:runner:ess": "npm run run-tests:rm prebuilt_rules/large_prebuilt_rules_package ess essEnv",
+    
+    "prebuilt_rules_update_prebuilt_rules_package:server:serverless": "npm run initialize-server:rm prebuilt_rules/update_prebuilt_rules_package serverless",
+    "prebuilt_rules_update_prebuilt_rules_package:runner:serverless": "npm run run-tests:rm prebuilt_rules/update_prebuilt_rules_package serverless serverlessEnv",
+    "prebuilt_rules_update_prebuilt_rules_package:qa:serverless": "npm run run-tests:rm prebuilt_rules/update_prebuilt_rules_package serverless qaEnv",
+    "prebuilt_rules_update_prebuilt_rules_package:server:ess": "npm run initialize-server:rm prebuilt_rules/update_prebuilt_rules_package ess",
+    "prebuilt_rules_update_prebuilt_rules_package:runner:ess": "npm run run-tests:rm prebuilt_rules/update_prebuilt_rules_package ess essEnv",
+    
+    "rule_delete:server:serverless": "npm run initialize-server:rm rule_delete serverless",
+    "rule_delete:runner:serverless": "npm run run-tests:rm rule_delete serverless serverlessEnv",
+    "rule_delete:qa:serverless": "npm run run-tests:rm rule_delete serverless qaEnv",
+    "rule_delete:server:ess": "npm run initialize-server:rm rule_delete ess",
+    "rule_delete:runner:ess": "npm run run-tests:rm rule_delete ess essEnv",
 
-    "rule_update:server:serverless": "npm run initialize-server:dr:default rule_update serverless",
-    "rule_update:runner:serverless": "npm run run-tests:dr:default rule_update serverless serverlessEnv",
-    "rule_update:qa:serverless": "npm run run-tests:dr:default rule_update serverless qaEnv",
-    "rule_update:server:ess": "npm run initialize-server:dr:default rule_update ess",
-    "rule_update:runner:ess": "npm run run-tests:dr:default rule_update ess essEnv",
+    "rule_import_export:server:serverless": "npm run initialize-server:rm rule_import_export serverless",
+    "rule_import_export:runner:serverless": "npm run run-tests:rm rule_import_export serverless serverlessEnv",
+    "rule_import_export:qa:serverless": "npm run run-tests:rm rule_import_export serverless qaEnv",
+    "rule_import_export:server:ess": "npm run initialize-server:rm rule_import_export ess",
+    "rule_import_export:runner:ess": "npm run run-tests:rm rule_import_export ess essEnv",
+    
```

Comment:
Nit: There is some trailing whitespace in a few of these files. GitHub seems not to care about it, but moving forward an editor setting like `"files.trimTrailingWhitespace": true` should address this automatically. cc @WafaaNasr

Reply:
Screenshot in case it's not clear what I mean: <img width="941" alt="Screenshot 2024-01-25 at 2 29 28 PM" src="https://github.com/elastic/kibana/assets/657252/7ff8487f-bd14-484c-8d11-1c130ffa1b7a">

Reply:
@rylnd Vscode on my end removes all the white spaces in the `package.json` so I had to keep them by myself, that's why there some extra my bad!!